week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -120,6 +131,9 @@ function getLocationName() { }
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
+  let found = locations.filter((p) => p.includes(transportMode));
+  return found.map(getLocationName);
+
   // Implement the function body
 }
```

Comment:
the last part of this function is wrong, in the instruction, it says return the first element of that filtered array